shared/src/business/utilities/pdfGenerator/components/SelectedFiltersSection.tsx

```diff
@@ -1,26 +1,34 @@
 import React from 'react';
 
-export const SelectedFiltersSection = ({ count, selectedFilters }) => (
+export const SelectedFiltersSection = ({ count, selectedFilters, trialStatusCounts }) => {  
+  return(
   <div className="card margin-top-0">
     <div className="card-header filters-selected-header">
       <div>Trial Status Filters Selected</div>
       <div className="align-right">Total Shown: {count}</div>
     </div>
     <div className="filters-selected-content">
       <div className="filters-row">
-        <div className="filter">{selectedFilters[0]?.label || ''}</div>
-        <div className="filter">{selectedFilters[2]?.label || ''}</div>
-        <div className="filter">{selectedFilters[4]?.label || ''}</div>
-        <div className="filter">{selectedFilters[6]?.label || ''}</div>
-        <div className="filter">{selectedFilters[8]?.label || ''}</div>
+        {[0, 2, 4, 6, 8].map(index => (
+          <div key={index} className="filter">
+            {selectedFilters[index]?.label || ''}
+            {selectedFilters[index]?.key in trialStatusCounts && (
+              <strong>({trialStatusCounts[selectedFilters[index].key]})</strong>
```

Comment:
Two nitpicks:

- I would recommend HTML for semantics and CSS classes for pure styling. `<strong>` _does_ change the styling, but it also signals "This is important!" In this case, because the count is not "important," I would recommend using a CSS class.
- It looks like there is no gap between the text and the parenthetical. I'm sure @katiecissell or @Evelyn-Durbin would catch this, but putting it here for fast feedback.

<img width="182" alt="Screenshot 2025-06-24 at 1 04 34 PM" src="https://github.com/user-attachments/assets/d363e87b-94c6-413d-ab45-1082e991ccfc" />